app/src/androidTest/java/net/skyscanner/backpack/flare/BpkFlareTest.kt

```diff
@@ -44,7 +44,7 @@ class BpkFlareTest : BpkSnapshotTest() {
 
   private val imageView = ImageView(testContext).apply {
     setImageDrawable(AppCompatResources.getDrawable(context, R.drawable.canadian_rockies_canada))
-    layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+    layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT)
```

Comment:
I'm honestly amazed that this worked before, but if the image is in the correct density folder we definitely need to be more strict about the view sizing